docs/lib/mdxUtils.ts

```diff
@@ -106,12 +108,18 @@ export function getPkgList(group?: string) {
 export function getPkgGroupList() {
 	return getPkgList().then((pkgs) => {
 		const uniqueGroups = new Map(pkgs.map((p) => [p.group, p.groupName]));
-		return Array.from(uniqueGroups.entries())
-			.map(([slug, title]) => ({
-				slug,
-				title,
-			}))
-			.sort((a, b) => (a.title > b.title ? 1 : -1));
+		return [
+			...PKG_GROUPS,
+			...Array.from(uniqueGroups.entries())
+				.map(([slug, title]) => ({
+					slug,
+					title,
+				}))
+				.filter(
+					(item) => !PKG_GROUPS.find((pkgGroup) => pkgGroup.slug === item.slug)
+				)
+				.sort((a, b) => (a.title > b.title ? 1 : -1)),
+		];
```

Comment:
I preferred the sorting solition. Primarily because it meant that there would be no confusion about how things in the Side Nav get their `title` and wether that aligns with other places the `title` is being displayed.

```suggestion
		return Array.from(uniqueGroups.entries())
			.map(([slug, title]) => ({
				slug,
				title,
			}))
			.sort((a, b) => {
				if (a.slug === 'foundations') return -1;
				return a.title > b.title ? 1 : -1;
			});
```